include/dlr_common.h

```diff
@@ -39,6 +39,17 @@
 #define DLR_DLL
 #endif  // defined(_MSC_VER) || defined(_WIN32)
 
+#ifndef DLR_MODEL_ELEM
+#define DLR_MODEL_ELEM
+enum DLRModelElemType { HEXAGON_LIB, NEO_METADATA, TVM_GRAPH, TVM_LIB, TVM_PARAMS, RELAY_EXEC };
+typedef struct ModelElem {
+   const DLRModelElemType type;
+   const char* path;
+   const void* data;
+   const size_t data_size;
+} DLRModelElem;
+#endif
```

Comment:
can we deduplicate this in dlr.h?

Reply:
In DLR we have few external C types and many internal C++ types. C++ backend does not see external C types. In rare cases internal and external declarations are the same. e.g. typedef `DLRMallocFunctionPtr`.  In that case we declare them in both dlr.h (external) and dlr_common.h (internal)